src/foundry/client/pixi/webgl/shaders/lighting/effects/flame.d.mts

```diff
@@ -0,0 +1,33 @@
+export {};
+
+declare global {
+  /**
+   * Alternative torch illumination shader
+   */
+  class FlameIlluminationShader extends AdaptiveIlluminationShader {
+    static override fragmentShader: string | ((...args: any[]) => string);
+
+    /**
+     * @defaultValue
+     * ```js
+     * {...super.defaultUniforms, brightnessPulse: 1}
+     * ```
+     */
+    static overridedefaultUniforms: AbstractBaseShader.Uniforms;
```

Comment:
This seems like a typo; you probably meant `static override defaultUniforms`.

Please also consider making a `FlameIlluminationShader.Uniforms` item.